### PR TITLE
Add IsEnableAppAuthPopUpEnabled to Set-SPOTenant

### PR DESCRIPTION
**Pull request by cmalonzo**

### CODE REVIEW

**Comment by cmalonzo** [comment]
Regarding this issue: https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/issues/407

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -2428,6 +2429,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -IsEnableAppAuthPopUpEnabled
+
+Enables or disables users in the tenancy to authenticate into SharePoint applications using popups.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2434]:
```suggestion
Enables or disables users in the organization to authenticate SharePoint applications using popups. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cmalonzo, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@cmalonzo Gentle reminder

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md (hunk 1 above)
Comment (lucabandMSFT) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2434]: This parameter affects the way code in SharePoint interacts with Azure AD to get token to access APIs. In scenarios where 3rd party cookies are disabled (e.g. Safari browsers with ITP feature enabled), any code that requires a token to access an API automatically triggers a full page refresh. When IsEnableAppAuthPopUpEnabled is set to $true, SharePoint will instead surface a popup if 3rd party cookies are disabled.

**Review by lucabandMSFT — CHANGES_REQUESTED** [review]
I would add a little bit of context to the admin to better understand what this parameters does. Let me know if that makes sense to you

**Comment by cmalonzo** [comment]
@lucabandMSFT, @JohanFreelancer9, and @yogkumgit  please rereview

**Review by lucabandMSFT — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 2]
```diff
@@ -2428,6 +2429,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -IsEnableAppAuthPopUpEnabled
+
+Enables or disables users in the organization to authenticate SharePoint applications using popups. 
+
+This parameter affects the way code in SharePoint interacts with Azure AD to get tokens to access APIs. In scenarios where third party cookies are disabled (e.g. Safari browsers with ITP feature enabled), any code that requires a token to access an API automatically triggers a full page refresh. When IsEnableAppAuthPopUpEnabled is set to $true, SharePoint will instead surface a popup in this scenario.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2436]:
```suggestion
This parameter affects the way code in SharePoint interacts with Azure AD to get tokens to access APIs. In scenarios where third-party cookies are disabled (such as Safari browsers with ITP feature enabled), any code that requires a token to access an API automatically triggers a full page refresh. When IsEnableAppAuthPopUpEnabled is set to $true, SharePoint will instead surface a popup in this scenario.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@cmalonzo, just a couple of minor suggestions, please.

**Comment by cmalonzo** [comment]
Thanks @JohanFreelancer9. Made changes and ready to merge.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**